lib/filepicker_rails/filepicker_image_url.rb

```diff
@@ -0,0 +1,141 @@
+# Creates the full path of the image to the specified `url` accepts optional
+# `options` hash for configuration.
+#
+# #### Constructor
+#
+# @param [String] url                       Base filepicker url
+# @param [Hash] options
```

Comment:
I received a error trying to genrate the docs:

```
[error]: Exception occurred while generating 'FilepickerRails.html'
[error]: NoMethodError: undefined method `parameters' for #<yardoc module FilepickerRails>
[error]: Stack trace:
	filepicker-rails/gems/yard-0.8.7.6/lib/yard/code_objects/base.rb:352:in `method_missing'
	filepicker-rails/gems/yard-0.8.7.6/templates/default/tags/html/option.erb:4:in `_erb_cache_33'
	filepicker-rails/gems/yard-0.8.7.6/lib/yard/templates/template.rb:279:in `erb'
	filepicker-rails/gems/yard-0.8.7.6/lib/yard/templates/template.rb:357:in `render_section'
	filepicker-rails/gems/yard-0.8.7.6/lib/yard/templates/template.rb:251:in `block (2 levels) in run'
	filepicker-rails/gems/yard-0.8.7.6/lib/yard/templates/template.rb:248:in `each'
```

Did you received any error? You can generate and open the docs with `yardoc && open doc/index.html`